docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Config_SmallRye.adoc

```diff
@@ -0,0 +1,155 @@
+[[MicroProfile_Config_SmallRye]]
+= MicroProfile Config Subsystem Configuration
+
+Support for https://microprofile.io/project/eclipse/microprofile-config[Eclipse MicroProfile Config] is provided by
+ the _microprofile-config-smallrye_ subsystem.
+
+[[required-extension]]
+== Required Extension
+
+This extension is included in the standard configurations included in the
+WildFly distribution.
+
+You can also add the extension to a configuration without it either by adding
```

Comment:
So MP config won't be enabled by default ?

Reply:
I mention just above that 
> This extension is included in the standard configurations included in the WildFly distribution

Is the wording ambiguous?